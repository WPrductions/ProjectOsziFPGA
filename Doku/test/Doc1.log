This is pdfTeX, Version 3.14159265-2.6-1.40.21 (TeX Live 2020/W32TeX) (preloaded format=pdflatex 2021.1.28)  31 JAN 2021 22:32
entering extended mode
 restricted \write18 enabled.
 %&-line parsing enabled.
**Doc1.tex
(./Doc1.tex
LaTeX2e <2020-10-01> patch level 4
L3 programming layer <2021-01-09> xparse <2020-03-03>
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/base/article.cls
Document Class: article 2020/04/10 v1.4m Standard LaTeX document class
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/base/size12.clo
File: size12.clo 2020/04/10 v1.4m Standard LaTeX file (size option)
)
\c@part=\count177
\c@section=\count178
\c@subsection=\count179
\c@subsubsection=\count180
\c@paragraph=\count181
\c@subparagraph=\count182
\c@figure=\count183
\c@table=\count184
\abovecaptionskip=\skip47
\belowcaptionskip=\skip48
\bibindent=\dimen138
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/graphics/color.sty
Package: color 2020/02/24 v1.2b Standard LaTeX Color (DPC)

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/graphics-cfg/color.cf
g
File: color.cfg 2016/01/02 v1.6 sample color configuration
)
Package color Info: Driver file: pdftex.def on input line 147.

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/graphics-def/pdftex.d
ef
File: pdftex.def 2020/10/05 v1.2a Graphics/color driver for pdftex
))
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/graphics/graphicx.sty
Package: graphicx 2020/09/09 v1.2b Enhanced LaTeX Graphics (DPC,SPQR)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/graphics/keyval.sty
Package: keyval 2014/10/28 v1.15 key=value parser (DPC)
\KV@toks@=\toks15
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/graphics/graphics.sty
Package: graphics 2020/08/30 v1.4c Standard LaTeX Graphics (DPC,SPQR)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/graphics/trig.sty
Package: trig 2016/01/03 v1.10 sin cos tan (DPC)
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/graphics-cfg/graphics
.cfg
File: graphics.cfg 2016/06/04 v1.11 sample graphics configuration
)
Package graphics Info: Driver file: pdftex.def on input line 105.
)
\Gin@req@height=\dimen139
\Gin@req@width=\dimen140
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/hyperref/hyperref.sty
Package: hyperref 2020-05-15 v7.00e Hypertext links for LaTeX

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/generic/ltxcmds/ltxcmds.sty
Package: ltxcmds 2020-05-10 v1.25 LaTeX kernel commands for general use (HO)
) (c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/generic/iftex/iftex.sty
Package: iftex 2020/03/06 v1.0d TeX engine tests
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/generic/pdftexcmds/pdftexcm
ds.sty
Package: pdftexcmds 2020-06-27 v0.33 Utility functions of pdfTeX for LuaTeX (HO
)

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/generic/infwarerr/infwarerr
.sty
Package: infwarerr 2019/12/03 v1.5 Providing info/warning/error messages (HO)
)
Package pdftexcmds Info: \pdf@primitive is available.
Package pdftexcmds Info: \pdf@ifprimitive is available.
Package pdftexcmds Info: \pdfdraftmode found.
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/generic/kvsetkeys/kvsetkeys
.sty
Package: kvsetkeys 2019/12/15 v1.18 Key value parser (HO)
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/generic/kvdefinekeys/kvdefi
nekeys.sty
Package: kvdefinekeys 2019-12-19 v1.6 Define keys (HO)
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/generic/pdfescape/pdfescape
.sty
Package: pdfescape 2019/12/09 v1.15 Implements pdfTeX's escape features (HO)
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/hycolor/hycolor.sty
Package: hycolor 2020-01-27 v1.10 Color options for hyperref/bookmark (HO)
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/letltxmacro/letltxmac
ro.sty
Package: letltxmacro 2019/12/03 v1.6 Let assignment for LaTeX macros (HO)
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/auxhook/auxhook.sty
Package: auxhook 2019-12-17 v1.6 Hooks for auxiliary files (HO)
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/kvoptions/kvoptions.s
ty
Package: kvoptions 2020-10-07 v3.14 Key value format for package options (HO)
)
\@linkdim=\dimen141
\Hy@linkcounter=\count185
\Hy@pagecounter=\count186

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/hyperref/pd1enc.def
File: pd1enc.def 2020-05-15 v7.00e Hyperref: PDFDocEncoding definition (HO)
Now handling font encoding PD1 ...
... no UTF-8 mapping file for font encoding PD1
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/generic/intcalc/intcalc.sty
Package: intcalc 2019/12/15 v1.3 Expandable calculations with integers (HO)
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/generic/etexcmds/etexcmds.s
ty
Package: etexcmds 2019/12/15 v1.7 Avoid name clashes with e-TeX commands (HO)
)
\Hy@SavedSpaceFactor=\count187
Package hyperref Info: Hyper figures OFF on input line 4464.
Package hyperref Info: Link nesting OFF on input line 4469.
Package hyperref Info: Hyper index ON on input line 4472.
Package hyperref Info: Plain pages OFF on input line 4479.
Package hyperref Info: Backreferencing OFF on input line 4484.
Package hyperref Info: Implicit mode ON; LaTeX internals redefined.
Package hyperref Info: Bookmarks ON on input line 4717.
\c@Hy@tempcnt=\count188
 (c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/url/url.sty
\Urlmuskip=\muskip16
Package: url 2013/09/16  ver 3.4  Verb mode for urls, etc.
)
LaTeX Info: Redefining \url on input line 5076.
\XeTeXLinkMargin=\dimen142

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/generic/bitset/bitset.sty
Package: bitset 2019/12/09 v1.3 Handle bit-vector datatype (HO)

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/generic/bigintcalc/bigintca
lc.sty
Package: bigintcalc 2019/12/15 v1.5 Expandable calculations on big integers (HO
)
))
\Fld@menulength=\count189
\Field@Width=\dimen143
\Fld@charsize=\dimen144
Package hyperref Info: Hyper figures OFF on input line 6347.
Package hyperref Info: Link nesting OFF on input line 6352.
Package hyperref Info: Hyper index ON on input line 6355.
Package hyperref Info: backreferencing OFF on input line 6362.
Package hyperref Info: Link coloring OFF on input line 6367.
Package hyperref Info: Link coloring with OCG OFF on input line 6372.
Package hyperref Info: PDF/A mode OFF on input line 6377.
LaTeX Info: Redefining \ref on input line 6417.
LaTeX Info: Redefining \pageref on input line 6421.

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/base/atbegshi-ltx.sty
Package: atbegshi-ltx 2020/08/17 v1.0a Emulation of the original atbegshi packa
ge
with kernel methods
)
\Hy@abspage=\count190
\c@Item=\count191
\c@Hfootnote=\count192
)
Package hyperref Info: Driver (autodetected): hpdftex.

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/hyperref/hpdftex.def
File: hpdftex.def 2020-05-15 v7.00e Hyperref driver for pdfTeX

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/base/atveryend-ltx.st
y
Package: atveryend-ltx 2020/08/19 v1.0a Emulation of the original atvery packag
e
with kernel methods
)
\Fld@listcount=\count193
\c@bookmark@seq@number=\count194

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/rerunfilecheck/rerunf
ilecheck.sty
Package: rerunfilecheck 2019/12/05 v1.9 Rerun checks for auxiliary files (HO)

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/generic/uniquecounter/uniqu
ecounter.sty
Package: uniquecounter 2019/12/15 v1.4 Provide unlimited unique counter (HO)
)
Package uniquecounter Info: New unique counter `rerunfilecheck' on input line 2
86.
)
\Hy@SectionHShift=\skip49
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/l3backend/l3backend-p
dftex.def
File: l3backend-pdftex.def 2021-01-09 L3 backend support: PDF output (pdfTeX)
\l__kernel_color_stack_int=\count195
\l__pdf_internal_box=\box47
) (./Doc1.aux)
\openout1 = `Doc1.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 5.
LaTeX Font Info:    ... okay on input line 5.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 5.
LaTeX Font Info:    ... okay on input line 5.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 5.
LaTeX Font Info:    ... okay on input line 5.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 5.
LaTeX Font Info:    ... okay on input line 5.
LaTeX Font Info:    Checking defaults for TS1/cmr/m/n on input line 5.
LaTeX Font Info:    ... okay on input line 5.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 5.
LaTeX Font Info:    ... okay on input line 5.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 5.
LaTeX Font Info:    ... okay on input line 5.
LaTeX Font Info:    Checking defaults for PD1/pdf/m/n on input line 5.
LaTeX Font Info:    ... okay on input line 5.

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/context/base/mkii/supp-pdf.
mkii
[Loading MPS to PDF converter (version 2006.09.02).]
\scratchcounter=\count196
\scratchdimen=\dimen145
\scratchbox=\box48
\nofMPsegments=\count197
\nofMParguments=\count198
\everyMPshowfont=\toks16
\MPscratchCnt=\count199
\MPscratchDim=\dimen146
\MPnumerator=\count266
\makeMPintoPDFobject=\count267
\everyMPtoPDFconversion=\toks17
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/epstopdf-pkg/epstopdf
-base.sty
Package: epstopdf-base 2020-01-24 v2.11 Base part for package epstopdf
Package epstopdf-base Info: Redefining graphics rule for `.eps' on input line 4
85.

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/latexconfig/epstopdf-
sys.cfg
File: epstopdf-sys.cfg 2010/07/13 v1.3 Configuration of (r)epstopdf for TeX Liv
e
))
Package hyperref Info: Link coloring OFF on input line 5.

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/hyperref/nameref.sty
Package: nameref 2019/09/16 v2.46 Cross-referencing by name of section

(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/latex/refcount/refcount.sty
Package: refcount 2019/12/15 v3.6 Data extraction from label references (HO)
)
(c:/Program Files (x86)/TexLive/2020/texmf-dist/tex/generic/gettitlestring/gett
itlestring.sty
Package: gettitlestring 2019/12/15 v1.6 Cleanup title references (HO)
)
\c@section@level=\count268
)
LaTeX Info: Redefining \ref on input line 5.
LaTeX Info: Redefining \pageref on input line 5.
LaTeX Info: Redefining \nameref on input line 5.
 (./Doc1.out) (./Doc1.out)
\@outlinefile=\write3
\openout3 = `Doc1.out'.

LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <14.4> on input line 11.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <7> on input line 11.
 (./Doc1.toc
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <12> on input line 3.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <8> on input line 3.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <6> on input line 3.
)
\tf@toc=\write4
\openout4 = `Doc1.toc'.

<Grafiken/ADC_Arduino_Pins.PNG, id=35, 958.17975pt x 431.211pt>
File: Grafiken/ADC_Arduino_Pins.PNG Graphic file (type png)
<use Grafiken/ADC_Arduino_Pins.PNG>
Package pdftex.def Info: Grafiken/ADC_Arduino_Pins.PNG  used on input line 25.
(pdftex.def)             Requested size: 426.79134pt x 192.06923pt.

Overfull \hbox (36.79134pt too wide) in paragraph at lines 25--26
 [] 
 []

<Grafiken/ADC_Front-End.PNG, id=39, 800.9925pt x 320.397pt>
File: Grafiken/ADC_Front-End.PNG Graphic file (type png)
<use Grafiken/ADC_Front-End.PNG>
Package pdftex.def Info: Grafiken/ADC_Front-End.PNG  used on input line 33.
(pdftex.def)             Requested size: 426.79134pt x 170.71892pt.

Overfull \hbox (36.79134pt too wide) in paragraph at lines 33--34
 [] 
 []


Underfull \hbox (badness 10000) in paragraph at lines 22--39

 []

[1

{c:/Program Files (x86)/TexLive/2020/texmf-var/fonts/map/pdftex/updmap/pdftex.m
ap}]

LaTeX Warning: `!h' float specifier changed to `!ht'.

[2 <./Grafiken/ADC_Arduino_Pins.PNG>]
Overfull \hbox (91.0373pt too wide) in paragraph at lines 51--53
[][]\OT1/cmr/m/n/12 Ubersicht: DE10-Lite-Board: DE10-Lite[]v.2.1.0[]SystemCD: D
E10-Lite[]User[]Manual.pdf 
 []


Overfull \hbox (16.3725pt too wide) in paragraph at lines 54--56
[]\OT1/cmr/m/n/12 Schaltplan: DE10-Lite-Board: DE10-Lite[]v.2.1.0[]SystemCD: de
10-lite.pdf 
 []

[3 <./Grafiken/ADC_Front-End.PNG>] (./Doc1.aux)
Package rerunfilecheck Info: File `Doc1.out' has not changed.
(rerunfilecheck)             Checksum: A6848A3B07ACEAC745E67FFAB5269BB3;382.
 ) 
Here is how much of TeX's memory you used:
 5243 strings out of 479304
 83459 string characters out of 5861285
 387804 words of memory out of 5000000
 22341 multiletter control sequences out of 15000+600000
 409451 words of font info for 47 fonts, out of 8000000 for 9000
 1141 hyphenation exceptions out of 8191
 71i,7n,74p,844b,372s stack positions out of 5000i,500n,10000p,200000b,80000s
{c:/Program Files (x86)/TexLive/2020/texmf-dist/fonts/enc/dvips/cm-super/cm-s
uper-ts1.enc}<c:/Program Files (x86)/TexLive/2020/texmf-dist/fonts/type1/public
/amsfonts/cm/cmbx12.pfb><c:/Program Files (x86)/TexLive/2020/texmf-dist/fonts/t
ype1/public/amsfonts/cm/cmr12.pfb><c:/Program Files (x86)/TexLive/2020/texmf-di
st/fonts/type1/public/amsfonts/cm/cmr17.pfb><c:/Program Files (x86)/TexLive/202
0/texmf-dist/fonts/type1/public/amsfonts/cm/cmti12.pfb><c:/Program Files (x86)/
TexLive/2020/texmf-dist/fonts/type1/public/amsfonts/cm/cmtt12.pfb><c:/Program F
iles (x86)/TexLive/2020/texmf-dist/fonts/type1/public/cm-super/sfrm1200.pfb>
Output written on Doc1.pdf (3 pages, 593790 bytes).
PDF statistics:
 94 PDF objects out of 1000 (max. 8388607)
 78 compressed objects within 1 object stream
 15 named destinations out of 1000 (max. 500000)
 59 words of extra memory for PDF output out of 10000 (max. 10000000)

