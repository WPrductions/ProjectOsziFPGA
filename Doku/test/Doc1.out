\BOOKMARK [1][-]{section.1}{Einleitung}{}% 1
\BOOKMARK [1][-]{section.2}{Entwicklung}{}% 2
\BOOKMARK [2][-]{subsection.2.1}{erste Schrittte}{section.2}% 3
\BOOKMARK [3][-]{subsubsection.2.1.1}{Verwendung der VGA Schnittstelle}{subsection.2.1}% 4
\BOOKMARK [3][-]{subsubsection.2.1.2}{Verwendung der ADCs}{subsection.2.1}% 5
\BOOKMARK [1][-]{section.3}{Quellen und Hilfsmittel}{}% 6
